documentary.md

```diff
@@ -15,59 +15,59 @@
 ## [Movies](movies.md)
 ## [Documentary](documentary.md)
 
-- [8.5 :star:](#_85)
+- [star:8.5 :](#_85)
```

Comment:
not sure about these changes... Now I suppose it will looks like `star: 8.5 :` in the web ui? But this change will broke GFM markup and will not show `star` symbol as ⭐️